src/Components/PublishablePositionCard/PublishablePositionCard.jsx

```diff
@@ -48,12 +55,14 @@ const PublishablePositionCard = ({
       { 'Position Number': data?.positionNumber || DEFAULT_TEXT },
       { 'Skill': data?.skill || DEFAULT_TEXT },
       { 'Position Title': data?.positionTitle || DEFAULT_TEXT },
+      { '': PP_FLAG() && <Link to="#" onClick={onRestore} >Restore</Link> },
     ],
     bodyPrimary: [
       { 'Bureau': data?.bureau || DEFAULT_TEXT },
       { 'Organization': data?.org || DEFAULT_TEXT },
-      { 'PP/Grade': data?.combined_pp_grade },
-      { 'Publishable Status': data?.status || DEFAULT_TEXT },
+      { 'Org Code': data?.orgCode || DEFAULT_TEXT },
+      { 'PP/Grade': data?.combined_pp_grade || DEFAULT_TEXT },
```

Comment:
```suggestion
      { 'PP/Grade': data?.combined_pp_grade },
```
`combined_pp_grade` already defaults text on the BE - it's one of the only places we do, cause of how many edge cases we have for Agendas (not that this is Agendas, but we did go ahead and standardize it)